langchain/schema.py

```diff
@@ -79,6 +79,8 @@ def type(self) -> str:
 class HumanMessage(BaseMessage):
     """Type of message that is spoken by the human."""
 
+    example: bool = False
```

Comment:
not sure notion of an example is generic enough to put on base abstraction. could we just pass in example message separately to `_messages_to_prompt_dict`?

Reply:
i think fine to keep as is - if we want to move to base class later it should be easy enough

Reply:
i was saying maybe we don't need to put it on any class at all, but don't feel strongly